api/handlers.go

```diff
@@ -211,6 +211,7 @@ func (api *API) loginPOSTToken(w http.ResponseWriter, req *http.Request) {
 		api.WriteError(w, err, http.StatusInternalServerError)
 		return
 	}
+	w.Header().Set("Skynet-Token", tokenStr)
```

Comment:
is this consistent naming? should it be `Skynet-JWT-Token`?

Reply:
I think it's fine. It's descriptive enough. I'm not sure we have an established convention for naming these things.